src/main/java/com/cmsc495/spacesim/model/Persons.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package com.cmsc495.spacesim.model;
+
+/**
+ *
+ * @author NoorQasim
+ */
+public class Persons {
+    
+    private String[] skill = {"Food", "Water", "Gasoline", "Coal", "Medical Supplies", "Builidng Materials"};
+    private String assignedPerson;
+    private String name;
+    private boolean locked;
+
+    
+    //add a skill
+    private void addSkill() {    

```

Comment:
This function should be public + static, and will add string values onto the `allPossibleSkills` array mentioned above.